addons/resource.language.en_gb/resources/strings.po

```diff
@@ -2860,59 +2868,59 @@ msgstr ""
 
 #: xbmc/settings/AdvancedSettings.cpp
 msgctxt "#669"
-msgid "Verbose logging for SMB library"
+msgid "Verbose logging for \"SMB\" library"
 msgstr ""
 
 #: xbmc/settings/AdvancedSettings.cpp
 msgctxt "#670"
-msgid "Verbose logging for CURL library (http, dav)"
+msgid "Verbose logging for \"libcURL\" library \"HTTP(S), DAV\""
```

Comment:
I would generally, agree with that, but Im looking at this from a general point of view how other entries are done, following @opendekamp suggestion to change CEC to libCEC

The sentence refers to **library**, and the library is libcurl.

Unless we reword all entries and use component instead of library but really wanted consistent entries and sentences throughout.